src/index.html

```diff
@@ -1,14 +1,55 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
```

Comment:
Remember about proper formatting
```suggestion
    <meta 
    	name="viewport"
```